plugins/mcreator-localization/lang/texts.properties

```
@@ -937,6 +941,7 @@ blockly.block.entity_vel_x=X velocity of %1 %2
 blockly.block.entity_vel_y=Y velocity of %1 %2
 blockly.block.entity_vel_z=Z velocity of %1 %2
 blockly.block.entity_xplevel=Get XP level of %1
+blockly.block.entity_xp_until_next_level=Get XP needed by %1 to gain a level
```

I think one of these is a bit more understandable:
```suggestion
blockly.block.entity_xp_until_next_level=Get amount of XP needed by %1 to gain a level
```
```suggestion
blockly.block.entity_xp_until_next_level=Get amount of XP needed by %1 to gain next level
```